examples/plot_pp_study.py

```diff
@@ -0,0 +1,362 @@
+constraint = DemographicParity()
+classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
+mitigator = ExponentiatedGradient(classifier, constraint)
+mitigator.fit(data_without_crimerate, y_true, sensitive_features=lstat_binned)
+y_pred_mitigated = mitigator.predict(data_without_crimerate)
+
+sr_mitigated = MetricFrame(selection_rate, y_true, y_pred_mitigated, sensitive_features=lstat_binned)
+print(sr_mitigated.overall)
+print(sr_mitigated.by_group)
+
+as_mitigated = MetricFrame(accuracy_score, y_true, y_pred_mitigated, sensitive_features=lstat_binned)
+print(as_mitigated.overall)
+print(as_mitigated.by_group)
+# %%
+#
+# By constraining selection rate disparity across LSTAT groups, we notice
+# a significant difference in the accuracy for the 3rd and 4th bins:
+#
+# **Before Mitigation - 95%+ accuracy on both bins**
+#
+#
+# **After Mitigation - 74% and 42% respectively**
+#
+#
+# The significant drop in accuracy indicates that the underlying issue may
+# not be at the model level but rather a reflection of the data (and
+# sociotechnical context itself). In the given data, LSTAT is a reliable
+# predictor of the crime rate threshold condition, which is a reflection
+# of systemic biases leading to this observed trend. This is why accuracy
+# for the higher bins falls off drastically if we try to achieve similar
+# selection rates across LSTAT bins.
+#
+# Problem Area & Sociotechnical Context
+# -------------------------------------
+#
+# The case study above highlights the importance of considering the
+# sociotechnical aspects of tasks traditionally deemed to be solely ML
+# problems. In this scenario, we’ve seen the grave consequences of failing
+# to acknowledge and ensure such considerations are a part of the design
+# and deployment of such ML systems. The result? A feedback loop of
+# discrimination and injustice due to lack of transparency and biased
+# training data. Some key takeaways for this case include:
+#
+#
+# * Algorithms / Algorithmic decision-making systems are not ‘objective’
+# * Lack of transparency and biased historical data leads to discriminatory models
+# * Biased data -> Biased predictions = Feedback loop of discrimination & injustice
+# * **Not all problems/tasks should be framed and approached as an ML task**
+#
+# For more background, check out this resource: `Predictive Policing
+# algorithms
+# <https://www.technologyreview.com/2020/07/17/1005396/predictive-policing-algorithms-racist-dismantled-machine-learning-bias-criminal-justice/>`_
+# are racist (MIT Technology Review)
+#
+# Conclusion
+# ----------
+#
+# The case study on predictive policing and applications of machine
+# learning techniques to making decision-making in the criminal justice
+# system more ‘objective’ reveals that this socio-technical problem
+# requires extensive thoughtful consideration. The findings show how
+# biased data and lack of transparency in models can result in ‘unfair’
```

Comment:
I think we should be drawing from the [fairness style guide](https://fairlearn.github.io/v0.5.0/contributor_guide/how_to_talk_about_fairness.html) when we talk about this piece. I also think these points about problems with objectivity and unfairness should be made more explicit earlier and throughout.